tests/tests_insights_remediation.yml

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Basic insights-client register/unregister
```

Comment:
needs a more fitting description ;)

Reply:
I suggest "Test for insights remediation register/unregister"